internal/cmd/oohelperd/measure.go

```diff
@@ -114,6 +117,49 @@ func measure(ctx context.Context, config *handler, creq *ctrlRequest) (*ctrlResp
 
 	// continue assembling the response
 	cresp.HTTPRequest = <-httpch
+
+	quicconnch := make(chan *quicResult, len(endpoints))
+
+	if cresp.HTTPRequest.DiscoveredH3 != "" {
+		// quicconnect: start over all the endpoints
+		for _, endpoint := range endpoints {
+			wg.Add(1)
+			go quicDo(ctx, &quicConfig{
+				Address:       endpoint.Addr,
+				Endpoint:      endpoint.Epnt,
+				Logger:        logger,
+				NewQUICDialer: config.NewQUICDialer,
+				URLHostname:   URL.Hostname(),
+				Out:           quicconnch,
+				Wg:            wg,
+			})
+		}
+		// http3: start
+		http3ch := make(chan ctrlHTTPResponse, 1)
+		wg.Add(1)
+
+		h3Client := func(logger model.Logger) model.HTTPClient {
+			reso := netxlite.MaybeWrapWithBogonResolver(
+				true, // enabled
+				newResolver(logger),
+			)
+			return netxlite.NewHTTP3ClientWithResolver(logger, reso)
+		}
+		go httpDo(ctx, &httpConfig{
+			Headers:           creq.HTTPRequestHeaders,
+			Logger:            logger,
+			MaxAcceptableBody: config.MaxAcceptableBody,
+			NewClient:         h3Client,
+			Out:               http3ch,
+			URL:               "https://" + cresp.HTTPRequest.DiscoveredH3,
+			Wg:                wg,
+			h3:                true,
```

Comment:
Please add to the above `httpConfig` for HTTP-over-TCP-or-TLS and explicit initialization of `h3` to `false`.

Reply:
renamed `h3` to `searchForH3` as you suggested above and set explicitly in both `httpConfig`s

Reply:
Great, thank you!